experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1257,61 +1257,121 @@ def test_no_treatment_branches_for_rollout(self):
             [NimbusConstants.ERROR_SINGLE_BRANCH_FOR_ROLLOUT],
         )
 
-    def test_rollout_valid_version_support(self):
-        desktop = NimbusExperiment.Application.DESKTOP
+    @parameterized.expand(
+        [
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MAC_ONLY,
+            ),
+            (
+                NimbusExperiment.Application.FENIX,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MAC_ONLY,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_ANDROID,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MAC_ONLY,
+            ),
+            (
+                NimbusExperiment.Application.IOS,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MAC_ONLY,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_IOS,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MAC_ONLY,
+            ),
+        ]
```

Comment:
you need to adjust targeting config according to the application, you have passed MAC_ONLY